API/Startup.cs

```diff
@@ -285,7 +297,17 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 o.OAuthClientId(Config.Swagger.ClientId);
             });
 
+            // using Microsoft.Extensions.FileProviders;
+            // using System.IO;
             app.UseStaticFiles();
+
+            app.UseFileServer(new FileServerOptions
+                              {
+                                  FileProvider = new PhysicalFileProvider(
+                                      Path.Combine(env.WebRootPath, "Resources")),
+                                  RequestPath = "/Resources",
+                                  EnableDirectoryBrowsing = true
```

Comment:
Is this needed? If so, why?

Reply:
I thought it was not working without it. Just tested and found out it is. So I removed it.